cmd/node/node.go

```diff
@@ -626,20 +626,14 @@ func (app *App) initServices(ctx context.Context,
 		return err
 	}
 	app.closers = append(app.closers, trtlStateDB)
-	trtlCfg := tortoise.Config{
-		LayerSize:       layerSize,
-		Database:        trtlStateDB,
-		MeshDatabase:    mdb,
-		ATXDB:           atxDB,
-		Clock:           clock,
-		Hdist:           app.Config.Hdist,
-		Zdist:           app.Config.Zdist,
-		ConfidenceParam: app.Config.ConfidenceParam,
-		WindowSize:      app.Config.WindowSize,
-		GlobalThreshold: app.Config.GlobalThreshold,
-		LocalThreshold:  app.Config.LocalThreshold,
-		Log:             app.addLogger(TrtlLogger, lg),
-		RerunInterval:   time.Minute * time.Duration(app.Config.TortoiseRerunInterval),
+	trtlCfg := tortoise.Configuration{
```

Comment:
i think that having both Configuration and Config just adds more technical debt. would be nicer if tortoise initializer was refactored to accept non-optional dependencies (atxdb, meshdb, etc) as parameters to function and keep only single Config for immutable parameters